README.md

```diff
@@ -54,11 +56,15 @@ To contribute to hassle free, follow these steps:
 
 Alternatively see the GitHub documentation on [creating a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
 
-
 ---
 
 ## Getting Started
 
+## Glimpse of UI
+
+**HOME**
+![Screenshot](../master/home.png)
+
```

Comment:
please add this png in PR comments to preview

Reply:
![home](https://user-images.githubusercontent.com/57443683/95090663-2632d180-0743-11eb-8a52-5556ed466211.png)